vector/src/main/java/im/vector/util/RoomUtils.java

```diff
@@ -510,18 +509,6 @@ public boolean onMenuItemClick(final MenuItem item) {
         } else {
             popup.getMenu().setGroupVisible(R.id.active_room_actions, true);
 
-            if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
```

Comment:
@Tyuoli Can you please keep all the code related to shortcut. I would just comment this block, 
and force `popup.getMenu().setGroupVisible(R.id.add_shortcut_actions, false);`